_site/public/tutorials/pet-shop.md

```diff
@@ -4,7 +4,7 @@
 
 # Ethereum Pet Shop
 
-This series of tutorials will take you through building your first Dapp--a tracking system for a pet shop!
+This series of tutorials will take you through building your first DApp--- a tracking system for a pet shop!
```

Comment:
Seems like markdown was making `--` into – (en dash), where that use warrants a — (em dash)